lib/utils/index.js

```diff
@@ -1024,6 +1024,18 @@ function download_zip_url(options = {}) {
   }));
 }
 
+/**
+ * Creates and downloads a URL that when invokes creates an archive of a folder
+ */
+function download_folder(folder_path, options = {}) {
+  options.resource_type = "all";
+  options.prefixes = folder_path;
```

Comment:
I wonder what's the correct approach if someone already has prefixes in options.
This is an array, right? could we add the `folder_path` param to it? what happens then, several folders are downloaded...?

Reply:
I understand you correctly.. you mean something like this? download_folder('folder1/', {prefixes: ['folder2/']});

In this case, only folder1 is downloaded. The prefixes option is ignored.

Reply:
yeah, that's what I meant. Not sure it's really important, just an edge case...